st3/lsp_utils/node_runtime.py

```diff
@@ -226,6 +240,13 @@ def npm_command(self) -> List[str]:
         return [self._npm]
 
 
+class NodeRuntimeAbsolute(NodeRuntime):
+    def __init__(self, node_binary: str):
+        super().__init__()
+        self._base_dir = path.abspath(path.dirname(node_binary))
+        self._node = path.join(self._base_dir, 'node')
+        self._npm = path.join(self._base_dir, 'npm')
```

Comment:
```suggestion
        self._npm = path.join(self._base_dir, 'npm')
        self._additional_paths = [path.dirname(self._node)]
```

this will fix the `node: No such file or directory` error

Reply:
Went ahead and pushed that change. Let me know if you're still experiencing problems.